lib/ProductOpener/Display.pm

```diff
@@ -7531,15 +7572,15 @@ JS
 				if (has_tag($product_ref, "labels", "en:palm-oil-free")
 					or ($ingredients_analysis_tag =~ /-free$/)) {
 					$ingredients_analysis_tag = "en:palm-oil-free";
-					$color = "#00aa00"; # green
+					$color = "#47a647"; # green
 					$icon = "monkey_happy";
```

Comment:
Does it fix #2929 ?

Reply:
Yes, someone changed the colors to be less saturated, but only for some fields.